cartography/data/jobs/cleanup/aws_import_instance_profiles_cleanup.json

```diff
@@ -0,0 +1,35 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (:CloudanixWorkspace{id: $WORKSPACE_ID})-[:OWNER]->(:AWSAccount{id: $AWS_ID})-[:RESOURCE]->(:AWSInstanceProfile)-[:HAS]->(n:AWSRole) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
we should not remove the role. that should be managed in iam specific cleanup file.